README.md

```diff
@@ -97,6 +97,14 @@ Examples:
 
 ### SELinux booleans
 
+#### allow_springboot_http_connect      (default: `true`)
+When switch to `true`this boolean allows the Springboot application to connect to remote
+HTTP/HTTPS ports (locally assigned the `http_port_t` SELinux type).
+
+#### allow_springboot_other_connect      (default: `true`)
+When switch to `true`this boolean allows the Springboot application to connect to remote
+other Springboot application ports (locally assigned the `springboot_port_t` SELinux type).
```

Comment:
to connect to remote other Springboot -> strange sentence

Reply:
allow_springboot_other_connect --> make me think it can connect to any other port :-(

Reply:
I couldn't figure out a more explicit yet short boolean name... still searching...